src/test/java/service/ReportServiceImplTest.java

```diff
@@ -0,0 +1,42 @@
+package service;
+
+import static org.junit.Assert.assertEquals;
+
+import dao.StorageDao;
+import dao.StorageDaoImpl;
+import db.Storage;
+import model.Fruit;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportServiceImplTest {
+    private static ReportService reportService;
+    private static StorageDao storageDao;
```

Comment:
Are we realy need `StorageDao` field in this class?
*Same qestion for every OperationHandlerTest